Dockerfile

```diff
@@ -1,12 +1,12 @@
 # registry.cloudogu.com/official/redmine
-FROM registry.cloudogu.com/official/base:3.5-5
-MAINTAINER Robert Auer <robert.auer@cloudogu.com>
+FROM registry.cloudogu.com/official/base:3.7-4
```

Comment:
If the versioning scheme is {remine-version}-{dogu-version} why is the docker version tag not:
- `3.4.6-4` 
- or (since I don't see the reason for the dogu version jump) `3.4.6-1`?

Reply:
Versioning in CES dogus/images is "TOOLVERSION"-"OUR_RELEASE_VERSION". If we release a dogu/image with a newer version of the tool, we start over with version 1.